pkg/enactmentstatus/conditions/counter_test.go

```diff
@@ -154,6 +162,35 @@ var _ = Describe("Enactment condition counter", func() {
 				failing:     CountByConditionStatus{t: 2, f: 0, u: 0},
 				progressing: CountByConditionStatus{t: 0, f: 2, u: 0},
 				matching:    CountByConditionStatus{t: 2, f: 0, u: 0},
+				aborted:     CountByConditionStatus{t: 0, f: 2, u: 0},
+			},
+		}),
+		Entry("e(Matching, Failed), e(Matching, Aborted)", EnactmentCounterCase{
+			policyGeneration: 1,
+			enactmentsToCount: enactments(
+				enactment(1, SetMatching, SetFailedToConfigure),
+				enactment(1, SetMatching, SetConfigurationAborted),
+			),
+			expectedCount: ConditionCount{
+				available:   CountByConditionStatus{t: 0, f: 2, u: 0},
+				failing:     CountByConditionStatus{t: 1, f: 1, u: 0},
+				progressing: CountByConditionStatus{t: 0, f: 2, u: 0},
+				matching:    CountByConditionStatus{t: 2, f: 0, u: 0},
+				aborted:     CountByConditionStatus{t: 1, f: 1, u: 0},
+			},
+		}),
+		Entry("e(Matching, Progressing), e(Matching, Aborted)", EnactmentCounterCase{
```

Comment:
Cannot be progressing + aborted I think we can remove this test.

Reply:
Done